GameData/RealismOverhaul/RO_SuggestedMods/Science818/Science818.cfg

```diff
@@ -0,0 +1,56 @@
+@PART[39A_pad]:FOR[RealismOverhaul]:AFTER[Science818]
+{
+	@rescaleFactor = 1.0
+    %RSSROConfig = True
+
+}
+@PART[39Base]:FOR[RealismOverhaul]:AFTER[Science818]
+{
+	@rescaleFactor = 1.0
+    %RSSROConfig = True
+}
+@PART[FSS]:FOR[RealismOverhaul]:AFTER[Science818]
+{
+	@rescaleFactor = 1.0
+    %RSSROConfig = True
+}
+@PART[FH_pad]:FOR[RealismOverhaul]:AFTER[Science818]
+{
+	@rescaleFactor = 1.0
+    %RSSROConfig = True
+}
+@PART[1.0_pad]:FOR[RealismOverhaul]:AFTER[Science818]
+{
+	@rescaleFactor = 1.0
+    %RSSROConfig = True
+}
+@PART[1.1_pad]:FOR[RealismOverhaul]:AFTER[Science818]
+{
+	@rescaleFactor = 1.0
+    %RSSROConfig = True
+}
+@PART[FT_pad]:FOR[RealismOverhaul]:AFTER[Science818]
+{
+	@rescaleFactor = 1.0
+    %RSSROConfig = True
+}
+@PART[LC40R]:FOR[RealismOverhaul]:AFTER[Science818]
+{
+	@rescaleFactor = 1.0
+    %RSSROConfig = True
+}
+@PART[F1_pad]:FOR[RealismOverhaul]:AFTER[Science818]
+{
+	@rescaleFactor = 1.0
+    %RSSROConfig = True
+}
+@PART[Vandy_F9]:FOR[RealismOverhaul]:AFTER[Science818]
+{
+	@rescaleFactor = 1.0
+    %RSSROConfig = True
+}
+@PART[Vandy_Heavy]:FOR[RealismOverhaul]:AFTER[Science818]
+{
+	@rescaleFactor = 1.0
+    %RSSROConfig = True
+} 
```

Comment:
It is not legal syntax to have 2 pass specifiers (FOR and AFTER).  Just pick one.  Most likely FOR[RealismOverhaul] is correct unless Science818 is doing something weird (like patching its own part's rescaleFactor in its own :FOR[Science818] pass....)

Also please fix the whitespace/indenting.

Reply:
I actually have to add AFTER because without it, the _RSSROConfig = True_ flag doesn't get applied.

Reply:
I've fixed the indexing and replaced the incorrect syntax with _:LAST_, since using _:FOR[REALISMOVERHAUL] won't work somehow.

Reply:
I think `LAST` requires a pass specifier.  `FIRST` doesn't, but `LAST` did.  `LAST[RealismOverhaul]`.
Can you clarify what you mean by "won't work?"  Did rescaleFactor not get set?  Does it have the wrong value?  Does `%rescaleFactor = 1.0` (`%` instead of `@`) work, ie if the field was not declared in the part?

Reply:
If you also had the ReScale patches from the Spacedock download, those edit rescaleFactor during the default pass.  This happens very early.  They would happen after your patches *if you didn't have any pass specifier at all* (and maybe if you have multiple/invalid specifiers) but a single specifier `FOR[RealismOverhaul]` should work just fine here.

Reply:
> I think `LAST` requires a pass specifier. `FIRST` doesn't, but `LAST` did. `LAST[RealismOverhaul]`.
> Can you clarify what you mean by "won't work?" Did rescaleFactor not get set? Does it have the wrong value? ~Does `%rescaleFactor = 1.0` (`%` instead of `@`) work, ie if the field was not declared in the part?~ (I went and looked at the original part config, this isn't it.)

the _%RSSROConfig = True_ part of the RO config did not get applied and so the parts appeared as "**Non-RO**"


> If you also had the ReScale patches from the Spacedock download, those edit rescaleFactor during the default pass. This happens very early. They would happen after your patches _if you didn't have any pass specifier at all_ (and maybe if you have multiple/invalid specifiers) but a single specifier `FOR[RealismOverhaul]` should work just fine here.

I did not have the SpaceDock rescale patches installed and for some weird reason _:FOR[REALISMOVERHAUL]_ didn't work in my testing, which is why I used _AFTER(Later corrected to LAST)_  however I will add REALISMOVERHAUL pass specifier after the LAST to it so it becomes :LAST[REALISMOVERHAUL]

Reply:
Please try it again with `:FOR[RealismOverhaul]` instead of LAST?  I cannot see why the FOR is incorrect, as that is literally when and how RO sets things everywhere.  There are a lot of things with exactly this structure.  <https://github.com/KSP-RO/RealismOverhaul/blob/master/GameData/RealismOverhaul/RO_SuggestedMods/SquadExpansion/RO_SquadExpansion_Structural.cfg> is nothing but this.

>the %RSSROConfig = True part of the RO config did not get applied and so the parts appeared as "Non-RO"

This suggests to me the entire patch simply did not run, period.  It doesn't appear that MM is case-sensitive, but maybe it is.  In which case, REALISMOVERHAUL in caps is never declared, and BEFORE/AFTER of a pass that is not declared will get removed.  (And you will see a note in the MM log about a pass being deleted because it did not meet its NEEDS)

Ex:
```
@PART[39A_pad]:FOR[RealismOverhaul]
{
    @rescaleFactor = 1.0
    %RSSROConfig = True
}
```

Reply:
No problem will correct to _FOR[RealismOverhaul]_  now.